### DISCUSSION TITLE
can we install two instances of noobaa in the same cluster with different namespace names

### DISCUSSION THREAD

**Discussion by vamseekrishna25** [noobaa/noobaa-core#6505] [Q&A]
can we install two instances of noobaa in the same cluster with different namespace names

**Comment by nimrod-becker** [comment]
Yes, you can install multiple instances on different namespaces.

Just notice that the CRDs NooBaa brings are cluster-wide, so in order to avoid collisions between versions its best to have all the instances with the same versions (or at least the same minor version)

**Reply by vamseekrishna25** [reply]
i wanted to install one instance of noobaa with 5.5.0-nsfs and other instance in a different project with version 5.6.0 or 5.7.0 is it possible as you told they should be same version is it that the second one should be 5.5.0.
 another doubt is if I delete one instance of noobaa is the other affected by it ?
 I tried to delete one instance of noobaa using the command noobaa uninstall --cleanup it deleted the other instance of noobaa also except the noobaa operator

**Comment by nimrod-becker** [comment]
--cleanup should probably get the namespace and clean only that namespace, but CRDs are cluster-wide so there might be a different issue.

**Reply by guymguym** [reply]
uninstall --cleanup will remove CRD's
https://github.com/noobaa/noobaa-operator/blob/ae8b269b3150d68e0965297e562898b186a03c89/pkg/install/install.go#L35
